hn.md

```diff
@@ -29,5 +29,4 @@ What to expect once you apply:
   * We will send you a 20-30 minute programming quiz
   * You will join 30 minute intro call and we will walk you through the compensation,
     interview process and requirements.
-  * You join a slack channel and submit a coding challenge in Go or Rust using GitHub.
-
+  * You join a Slack channel and submit a coding challenge in Go using GitHub.
```

Comment:
Should it say Rust?

Reply:
No, this is the generic ad we occasionally post to HN for the standard systems role. We require Go for that one, Rust is only for the Windows team.